source/develop/troubleshooting-nfs-storage-issues.md

```diff
@@ -147,6 +147,37 @@ A new **nfs-check** script is now available to test whether an NFS export is rea
       # systemctl start rpcbind 
       # systemctl start nfs-server
 
+#### Synology DSM 7
```

Comment:
Blank line after a heading makes the source easier to read.

Reply:
Added